pkg/acsengine/engine.go

```diff
@@ -635,7 +635,8 @@ func getPackageGUID(orchestratorType api.OrchestratorType, masterCount int) stri
 func getDCOSCustomDataPublicIPStr(orchestratorType api.OrchestratorType, masterCount int) string {
 	if orchestratorType == api.DCOS173 ||
 		orchestratorType == api.DCOS184 ||
-		orchestratorType == api.DCOS187 {
+		orchestratorType == api.DCOS187 ||
+		orchestratorType == api.DCOS188 {
```

Comment:
Not as far as we can tell from diffing the 1.8.7 and 1.8.8 templates from Mesosphere.